extra/resources/ClusterMon.in

```diff
@@ -69,15 +69,15 @@ Additional options to pass to crm_mon.  Eg. -n -r
 <content type="string" default="" />
 </parameter>
 
-<parameter name="pidfile">
+<parameter name="pidfile" unique-group="htmlfile">
 <longdesc lang="en">
 PID file location to ensure only one instance is running
 </longdesc>
 <shortdesc lang="en">PID file</shortdesc>
 <content type="string" default="/tmp/ClusterMon_${OCF_RESOURCE_INSTANCE}.pid" />
 </parameter>
 
-<parameter name="htmlfile" required="0">
+<parameter name="htmlfile" unique-group="pidfile" required="0">
```

Comment:
You flipped the names for unique-group. That would actually work, since it just needs to be any unique name, but it would confuse the heck out of people :)